src/internal/views/nunjucks/contact-entry/basic-form.njk

```diff
@@ -0,0 +1,15 @@
+{% extends "./nunjucks/layout.njk" %}
```

Comment:
No need to create a new one you can just use the shared basic form src/internal/views/nunjucks/form.njk